crates/router/src/core/errors/api_error_response.rs

```diff
@@ -271,6 +273,9 @@ impl common_utils::errors::ErrorSwitch<api_models::errors::types::ApiErrorRespon
                 1,
                 "API key not provided or invalid API key used", None
             )),
+                Self::GenericNotFoundError { message } => {
+                AER::NotFound(ApiError::new("HE", 5, format!("{message:?}"), None))
```

Comment:
If `HE_05` is the intended error code, can you keep it along with the rest of the `HE_*` errors?